LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -40,6 +40,7 @@ extern "C" {
 #endif
 
 using namespace std;
+using namespace fleece;
```

Comment:
It looks like there are no real changes in this file anymore, so best to just reset it.